packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/Line.java

```diff
@@ -61,6 +67,18 @@ public List<ICell> getCells() {
         return this.cells;
     }
 
+    public Integer getInitialHeight() {
+        return this.initialHeight;
+    }
+
+    public Integer getHeight() {
+        return this.height;
+    }
```

Comment:
Same comment and unless I missed something, you don't need both values. Just the height.

Reply:
Done